modules/sdk-coin-zketh/src/zkethToken.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @prettier
+ */
+import { EthLikeTokenConfig } from '@bitgo/statics';
+import { BitGoBase, CoinConstructor, NamedCoinConstructor } from '@bitgo/sdk-core';
+import { CoinNames, EthLikeToken } from '@bitgo/abstract-eth';
+
+export { EthLikeTokenConfig };
+
+export class ZkethToken extends EthLikeToken {
+  public readonly tokenConfig: EthLikeTokenConfig;
+  static coinNames: CoinNames = {
+    Mainnet: 'zketh',
+    Testnet: 'tzketh',
+  };
+  constructor(bitgo: BitGoBase, tokenConfig: EthLikeTokenConfig) {
+    super(bitgo, tokenConfig, ZkethToken.coinNames);
+  }
+  static createTokenConstructor(config: EthLikeTokenConfig): CoinConstructor {
+    return super.createTokenConstructor(config, ZkethToken.coinNames);
+  }
+
+  static createTokenConstructors(): NamedCoinConstructor[] {
+    return super.createTokenConstructors(ZkethToken.coinNames);
+  }
```

Comment:
I think getTransactionBuilder method will have to be added here as it is not there in the parent classes. It can be added in a follow up PR

Reply:
Added.